code/modules/hydroponics/grown/grass_carpet.dm

```diff
@@ -131,3 +131,18 @@
 	filling_color = "#245c39"
 	bitesize_mod = 3
 	can_distill = FALSE
+	var/value = 0
```

Comment:
If you wanted a luck stat system, avoid using an unclear name like 'value'
If the variable will not be used right now, don't declare it because it will make other coders bothered to see 'this var is not used' error message